src/main/java/mate/academy/springboot/swagger/service/Sorter.java

```diff
@@ -0,0 +1,34 @@
+package mate.academy.springboot.swagger.service;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.springframework.data.domain.Sort;
+import org.springframework.stereotype.Component;
+
+@Component
+public class Sorter {
+    public Sort getSortParams(String sortBy) {
+        List<Sort.Order> orders = new ArrayList<>();
+        // IDK why we need this part...        if (sortBy.contains(":")) {
+        String[] sortingFields = sortBy.split(";");
+        for (String field : sortingFields) {
+            Sort.Order order;
+            if (field.contains(":")) {

```

Comment:
It works. But why do we need to do splitting and go into the loop when it is not a semicolon? 
Also, you can replace the hardcode in the method with constants.

Reply:
Agree with applying constants. But I anyway check if semicolons exist below. By this check ` if (sortBy.contains(":"))`, we presume that if the request doesn't contain `:`, it also doesn`t contain `;`, so request like `/products?sortBy=title;price` won`t work.